hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/handlers/end/ExtraHeadersExtractor.java

```diff
@@ -0,0 +1,22 @@
+package pl.allegro.tech.hermes.frontend.publishing.handlers.end;
+
+import io.undertow.server.HttpServerExchange;
+import pl.allegro.tech.hermes.frontend.publishing.metadata.HeadersPropagator;
+
+import javax.inject.Inject;
+import java.util.Map;
+
+import static pl.allegro.tech.hermes.frontend.publishing.metadata.HeadersToMapTransformer.toHeadersMap;
+
+public class ExtraHeadersExtractor {
```

Comment:
This class is used only in the context of logging... so maybe we should change the class name to be more accurate ? Just like `TracingHeadersExtractor` ?